flickit-assessment-kit/src/test/java/org/flickit/assessment/kit/application/service/questionimpact/CreateQuestionImpactServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package org.flickit.assessment.kit.application.service.questionimpact;
+
+import org.flickit.assessment.common.exception.AccessDeniedException;
+import org.flickit.assessment.common.exception.ResourceNotFoundException;
+import org.flickit.assessment.kit.application.domain.QuestionImpact;
+import org.flickit.assessment.kit.application.port.in.questionimpact.CreateQuestionImpactUseCase;
+import org.flickit.assessment.kit.application.port.out.expertgroup.LoadExpertGroupOwnerPort;
+import org.flickit.assessment.kit.application.port.out.kitversion.LoadKitVersionPort;
+import org.flickit.assessment.kit.application.port.out.questionimpact.CreateQuestionImpactPort;
+import org.flickit.assessment.kit.test.fixture.application.AssessmentKitMother;
+import org.flickit.assessment.kit.test.fixture.application.KitVersionMother;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_NOT_ALLOWED;
+import static org.flickit.assessment.kit.common.ErrorMessageKey.KIT_VERSION_ID_NOT_FOUND;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CreateQuestionImpactServiceTest {
+
+    @InjectMocks
+    private CreateQuestionImpactService service;
+
+    @Mock
+    private LoadKitVersionPort loadKitVersionPort;
+
+    @Mock
+    private LoadExpertGroupOwnerPort loadExpertGroupOwnerPort;
+
+    @Mock
+    private CreateQuestionImpactPort createQuestionImpactPort;
+
+    @Test
+    void testCreateQuestionImpactServiceTest_kitVersionIdDoesNotExist_throwsResourceNotFoundException() {
```

Comment:
rename test names prefix from `testCreateQuestionImpactServiceTest` to `testCreateQuestionImpact`